Acornfile

```diff
@@ -11,12 +11,15 @@ args: {
   // Cloud provider region
   region: "US_EAST_1"
   
-  // Cluster size (only M0 is free)
-  tier: "M0"
+  // Atlas project name
+  projectName: "mytestproject"
```

Comment:
Will this allow the user to use an existing project? I haven't used atlas much, but is it typical to create multiple DBs per project? Is that how access is granted to users/teams?

Reply:
As a user can already have a M0 cluster in its project, I think creating a new project can be interesting. Yes, access is granted at the project level.